backend/infrahub/core/diff/merger/model.py

```diff
@@ -0,0 +1,19 @@
+from typing_extensions import TypedDict
```

Comment:
I think we perhaps have this in the SDK for backward compatible reasons and probably somewhere within the core as well but with the version of Python we're running now we can use `from typing import TypedDict` and eventually phase out the requirement to install `typing_extensions`